src/transformers/models/x_clip/modeling_x_clip.py

```diff
@@ -1218,8 +1218,8 @@ def __init__(self, config):
 
         embed_dim = config.projection_dim
         self.cross_attn = XCLIPCrossAttention(config)
-        self.norm1 = nn.LayerNorm(embed_dim)
-        self.norm3 = nn.LayerNorm(embed_dim)
+        self.norm1 = nn.LayerNorm(embed_dim, eps=config.text_config.layer_norm_eps)
```

Comment:
For the change here and below, I tried to read the source code and use the `text_config` and `vision_config` depending on the context. (Although, if `eps` gets the same value in these 2 configs, either could work.)

cc @NielsRogge to see if you have any comment.